socorro/external/fs/crashstorage.py

```diff
@@ -639,7 +639,8 @@ def _get_base(self, crash_id):
 FSDatedPermanentStorage = FSLegacyDatedRadixTreeStorage
 
 
-class TarFileCrashStore(CrashStorageBase):
+#==============================================================================
+class TarFileWritingCrashStore(CrashStorageBase):
```

Comment:
Either the name is misleading or we need a doc string here. 
After all, it takes the gzips the processed crash to disk.